google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/it/WriteRetryTestUtil.java

```diff
@@ -147,6 +147,7 @@ public static void runDefaultRetryTest(
         JsonStreamWriter.newBuilder(parent.toString(), tableSchema)
             .setIgnoreUnknownFields(true)
             .setRetrySettings(retrySettings)
+            .setEnableConnectionPool(true)
```

Comment:
Could we have on test for non multiplexed run and one for multiplexed run?

Reply:
Done